### PR TITLE
OCPBUGS-26494: PowerVS: handle 4.16 changes

### PR DESCRIPTION
**Pull request by hamzy**
1) Change the --bind-address argument to 127.0.0.1
2) Remove updating the pod IP address

### CODE REVIEW

**Review by mkumatag — APPROVED** [review]
/lgtm

File: pkg/cloud/powervs/assets/deployment.yaml [hunk 1]
```diff
@@ -77,7 +73,7 @@ spec:
               source /etc/kubernetes/apiserver-url.env
             fi
             exec /bin/ibm-cloud-controller-manager \
-            --bind-address=$(POD_IP_ADDRESS) \
+            --bind-address=$(127.0.0.1) \
```
Comment (JoelSpeed) [inline pkg/cloud/powervs/assets/deployment.yaml:76]: This is a literal not a variable
```suggestion
            --bind-address=127.0.0.1 \
```

Comment (hamzy) [reply pkg/cloud/powervs/assets/deployment.yaml]: stupid mistake!

**Comment by JoelSpeed** [comment]
Change looks ok but I'd like to see an optional E2E job on the repo that proves this works

**Comment by hamzy** [comment]
> Change looks ok but I'd like to see an optional E2E job on the repo that proves this works

I have hand edited a deploying cluster on the original changes and they worked for me.

**Comment by hamzy** [comment]
I will also add that we should eventually work to get that testing done.  However, the PowerVS CI is not stable enough at the moment to require that testing before this PR merges.

**Comment by JoelSpeed** [comment]
> I will also add that we should eventually work to get that testing done. However, the PowerVS CI is not stable enough at the moment to require that testing before this PR merges.

Not even stable enough to prove that a cluster can at least install? Given we know this prevents installs?

**Comment by hamzy** [comment]
> > I will also add that we should eventually work to get that testing done. However, the PowerVS CI is not stable enough at the moment to require that testing before this PR merges.
> 
> Not even stable enough to prove that a cluster can at least install? Given we know this prevents installs?

Well it is more stable to get past the cluster installation step (I would guess > 80 or 90%).  It is getting past running E2E testing where passing every test case consistently is problematic.

**Comment by hamzy** [comment]
I guess the optional passing requirement would at least not block other arches.

**Comment by JoelSpeed** [comment]
What I'm interested in for the purpose of platforms like PowerVS is being able to check the CCM logs on CI runs and see if the CCM is doing what we expect it to do. Which I expect you can probably do with the current state of PowerVS CI

**Pull request merged**